src/rust/ensogl/lib/core/Cargo.toml

```diff
@@ -41,9 +40,15 @@ num-traits = { version = "0.2" }
 paste = { version = "0.1.6" }
 rustc-hash = { version = "1.0.1" }
 shrinkwraprs = { version = "0.3.0" }
-smallvec = { version = "1.0.0" }
+smallvec = { version = "1.6.1" }
 typenum = { version = "1.11.2" }
 
+# The syn dependency is only here to lock the version used by `enum_dispatch`. Otherwise an
+# incompatible version will be chosen and compilation fails. This can be removed when
+#`enum_dispatch` has released a working version and we can update to that.
```

Comment:
Isthere a ticket for it in enum-dispatch ? If not, can we create it and link here? Otherwise its impossible to track it down in the future.

Reply:
This is now fixed through an update of `anyhow`.